src/data/endpoints/account.yaml

```diff
@@ -53,6 +53,23 @@ endpoints:
             my_settings.country = 'US'
             my_settings.email = 'jsmith@company.com'
             my_settings.save()
+  /account/settings/managed-enable:
+    group: Settings
+    authenticated: true
+    description: >
+      Enable Linode Managed on youy account.
+    methods:
+      POST:
+        money: true
+        scopes: account:modify
+        description: >
+          Enable Linode Managed on youy account. Only unrestricted users can do this.
+          See https://linode.com/managed for details and pricing information.
```

Comment:
I would really like to make this a link, but I couldn't figure out how to/find an example of doing so.  Anyone know how to do this? `<ExternalLink to="">` didn't work, `<a href="">` didn't work, and a `seeAlso`didn't do what I wanted.